tests/functional_tests/serialization_test.py

```diff
@@ -38,10 +38,7 @@ def electron_function(x):
 
 
 @etron
-@cova.lattice(
-    results_dir="./",
-    dispatcher="super long fake dispatcher to test really long arguments                                                                                                                                                                                end_fake_dispatcher_name",
-)
+@cova.lattice(results_dir="./")
```

Comment:
I'm little confused about import covalent and then covalent again as cova.

Reply:
I think these changes were introduced in a separate PR done before I started working on this PR. As for the reason, I think @jkanem did this in order to test that things still work if someone imports `covalent` as `cova`.